doc/nrf/known_issues.rst

```diff
@@ -166,12 +166,29 @@ Missing :file:`CMakeLists.txt`
 Thread
 ======
 
-.. rst-class:: v1-4-0
+.. rst-class:: v1-4-99-dev1
+
+KRKNWK-8262: CoAP Client Sample crashes for nRF5340
+  For nRF5340, the :ref:`coap_client_sample` sample crashes when changing from SED to MED mode.
+  Other devices are not affected.
+  The crash is caused by mode switching and IPC communication being handled by the system workqueue, which causes a deadlock.
+
+  **Workaround:** Use the application default thread, a dedicated workqueue, or a new thread to handle operations that require radio interactions.
+  If you cannot avoid using the system workqueue, cherry-pick the upstream `Zephyr commit f12536`_.
+
+  .. rst-class:: v1-4-99-dev1
+
+KRKNWK-8291: CoAP Client and Server samples do not use Hardware Flow Control on serial port
+  Hardware Flow Control on the serial port was disabled for the :ref:`coap_client_sample` and :ref:`coap_server_sample` Samples, but this change was not reflected in the documentation.
```

Comment:
There is a PR updating the documentation:
https://github.com/nrfconnect/sdk-nrf/pull/3455

If it gets merged soon maybe we should skip `but this change was not reflected in the documentation.`

Reply:
I don't think this PR will go into the 1.4.99_dev1 release.